ruby/private/utils.bzl

```diff
@@ -139,3 +139,27 @@ def normalize_bzlmod_repository_name(name):
         repository name
     """
     return name.rpartition("~")[-1]
+
+def is_version_older(current_version, minimum_version):
```

Comment:
I generally use https://github.com/bazelbuild/bazel-skylib/blob/main/lib/versions.bzl for this - we do have a skylib dependency already. Did you consider it already?

Reply:
No, I wasn't aware of this, let me try to use it instead.

Reply:
yes, this is much better, please take another look